lmms_eval/api/metrics.py

```diff
@@ -405,7 +405,10 @@ def gpt4judge(references, predictions, query):  # This is a passthrough function
                     eval_logger.error(f"All 5 attempts failed. Last error message: {str(e)}.\nResponse: {str(error_msg)}")
                     response = ""
 
-        score = int(extract_number_from_brackets(response))
+        if response is None:  # Rare case of gpt returning empty response
+            score = 0
```

Comment:
Can we do a try+except wrapping `extract_number_from_brackets`, and also add logging whenever this is caught?